src/platform/user/profile/vet360/containers/Vet360ProfileField.jsx

```diff
@@ -108,7 +108,10 @@ class Vet360ProfileField extends React.Component {
 
     const method = payload.id ? 'PUT' : 'POST';
 
-    if (this.props.useAddressValidation) {
+    if (
+      this.props.fieldName.toLowerCase().includes('address') &&
```

Comment:
This bit was accidentally removed here https://github.com/department-of-veterans-affairs/vets-website/pull/11740/files#diff-6f6cfbacf7aae6e221ef96d53de1bb31R111